### PR TITLE
8290391: Reduce runtime of java.util package microbenchmarks

### PR DESCRIPTION
**Pull request by ericcaspole**
Adds Warmup, Measurement and Fork annotations to reduce the run time from 1 day 16 hours to 8 hours for this set.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8290391](https://bugs.openjdk.org/browse/JDK-8290391): Reduce runtime of java.util package microbenchmarks

### Reviewers
 * [Roger Riggs](https://openjdk.org/census#rriggs) (@RogerRiggs - **Reviewer**) ⚠️ Review applies to [8d71eb4e](https://git.openjdk.org/jdk/pull/9523/files/8d71eb4eef0324592aa6a89bf45858e0906750da)
 * [Claes Redestad](https://openjdk.org/census#redestad) (@cl4es - **Reviewer**) ⚠️ Review applies to [8d71eb4e](https://git.openjdk.org/jdk/pull/9523/files/8d71eb4eef0324592aa6a89bf45858e0906750da)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9523/head:pull/9523` \
`$ git checkout pull/9523`

Update a local copy of the PR: \
`$ git checkout pull/9523` \
`$ git pull https://git.openjdk.org/jdk pull/9523/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9523`

View PR using the GUI difftool: \
`$ git pr show -t 9523`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9523.diff">https://git.openjdk.org/jdk/pull/9523.diff</a>

</details>

### CODE REVIEW

**Review by RogerRiggs — APPROVED** [review]
LGTM

File: test/micro/org/openjdk/bench/java/util/ArraysMismatchPartialInlining.java [hunk 1]
```diff
@@ -38,6 +39,9 @@
 @BenchmarkMode(Mode.Throughput)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @State(Scope.Thread)
+@Warmup(iterations = 4, time = 2)
+@Measurement(iterations = 4, time = 2)
+@Fork(value = 3)
 public class ArraysMismatchPartialInlining {
 
     @Param({"3", "4", "5", "6", "7", "15", "31", "63", "95", "800"})
```
Comment (cl4es) [inline test/micro/org/openjdk/bench/java/util/ArraysMismatchPartialInlining.java:47]: Can we prune a few of these? At least 4, 5, 6 seem like they'll not add much signal.

File: test/micro/org/openjdk/bench/java/util/Base64VarLenDecode.java [hunk 2]
```diff
@@ -85,6 +84,9 @@ public void doTearDownInvocation() {
     }
 
     @Benchmark
+    @Warmup(iterations = 4, time = 2)
```
Comment (cl4es) [inline test/micro/org/openjdk/bench/java/util/Base64VarLenDecode.java:87]: I prefer putting these at class level (even when there's only one `@Benchmark`)

**Review by cl4es — APPROVED** [review]
LGTM, some suggestions inline that you can choose to ignore

**Pull request closed without merging**